src/Scene/NodeProcess.js

```diff
@@ -324,18 +322,9 @@ function updateNodeElevation(quadtree, node, layersConfig, force) {
     // We don't care about layer status (isLayerFrozen) or limits (tileInsideLimit) because
     // we simply want to use ancestor's texture with a different pitch
     if (currentElevation == -1) {
-        for (let i = 0; i < elevationLayers.length; i++) {
-            let layer = elevationLayers[i];
-
-            if (currentElevation === -1) {
-                let a = findAncestorWithValidTextureForLayer(node, l_ELEVATION, layer);
-
-                if (a != null) {
-                    bestLayer = layer;
-                    ancestor = a;
-                    break;
-                }
-            }
+        let a = findAncestorWithValidTextureForLayer(node, l_ELEVATION);
```

Comment:
bestLayer was supposed to be set at the same time than the ancestor. Or the 1st pass thing is useless (it's supposed to retrieve ancestor inital texture)